src-backend/api/tests/test_pages.py

```diff
@@ -0,0 +1,107 @@
+from django.test import TestCase, Client, SimpleTestCase
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from nose.tools import assert_equals
+from api.models import User, Page, Procedure
+from utils.decorators import add_group_permissions
+import json
+
+
+class PageTest(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+        self.user = User.objects.create_user(
```

Comment:
Can you use factory boy here. If the values are irrelevant (besides associating pages and such, it'll auto-generate them.